add_to_app/MultipleFluttersIos/MultipleFluttersIos/DoubleFlutterViewController.swift

```diff
@@ -0,0 +1,18 @@
+import UIKit
+
+class DoubleFlutterViewController : UIViewController {
```

Comment:
I don't mean specifically in this arrangement either, but we should find some way to show this off a bit more. Since we can handle something like 5 Flutters in a screen.

Reply:
That isn't what we discussed doing.  It doesn't really provide a sample of something that doesn't exist and as the screens get smaller you won't even be able to press the buttons.